web/src/components/SharingPopup.tsx

```diff
@@ -0,0 +1,258 @@
+import React, { ReactElement, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled, { css } from 'styled-components'
+
+import { getSlugFromPath, POIS_ROUTE } from 'api-client'
+import { UiDirectionType } from 'translations'
+
+import { CloseIcon, FacebookIcon, MailSocialIcon, ShareActiveIcon, ShareIcon, WhatsappIcon } from '../assets'
+import dimensions from '../constants/dimensions'
+import useWindowDimensions from '../hooks/useWindowDimensions'
+import { RouteType } from '../routes'
+import Portal from './Portal'
+import ToolbarItem from './ToolbarItem'
+import Tooltip from './Tooltip'
+
+type SharingPopupProps = {
+  shareUrl: string
+  title: string
+  flow: 'vertical' | 'horizontal'
+  direction: UiDirectionType
+  route: RouteType
+}
+
+const TooltipContainer = styled.div<{
+  flow: 'vertical' | 'horizontal'
+  active: boolean
+  direction: UiDirectionType
+  additionalPadding: number
+  horizontalPosition?: number
+}>`
+  background-color: ${props => props.theme.colors.backgroundColor};
+  padding: 8px;
+  border: 2px solid ${props => props.theme.colors.textDecorationColor};
+  width: max-content;
+  position: absolute;
+  display: flex;
+  z-index: 2000;
+  opacity: 0;
+
+  ${props =>
+    props.flow === 'vertical' &&
+    css`
+      flex-flow: column-reverse;
+      transform: translateY(-100%);
+    `};
+
+  ${props =>
+    props.flow === 'horizontal' &&
+    (props.direction === 'ltr'
+      ? css`
+          transform: translateX(30%);
+        `
+      : css`
+          transform: translate(-30%);
+        `)};
+
+  ${props =>
+    props.active &&
+    css`
+      animation: tooltips 300ms ease-out forwards;
+    `};
+
+  &:before,
+  &:after {
+    position: absolute;
+    content: '';
+    display: block;
+  }
+
+  &:before {
+    z-index: 2000;
+    border-bottom: 10px solid ${props => props.theme.colors.backgroundColor};
+    border-left: 10px solid transparent;
+    border-right: 10px solid transparent;
+
+    ${props =>
+      props.flow === 'vertical' &&
+      (props.direction === 'ltr'
+        ? css`
+            left: 20px;
```

Comment:
Don't we then need `right: 20px;` for RTL layouts below?

Reply:
hm there is right missing true. thats because of the padding. Added even lower value. not sure why